packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -41,6 +41,8 @@ contract StableToken is
 
   event TransferComment(string comment);
 
+  bytes32 constant GRANDA_MENTO_REGISTRY_ID = keccak256(abi.encodePacked("GrandaMento"));
```

Comment:
though this is already a pervasive problem, this is another departure from modularity in the contracts
a whitelist of identifiers in storage is the alternative

Reply:
Yeah kinda torn on this one. I agree a whitelist of identifiers is much nicer but I'm not sure if it's appropriate for Granda Mento to also navigate that change given it'll expand the surface area a lot & make the contract release more complicated. Wdyt?